apps/backend/src/modules/auth/auth.service.ts

```diff
@@ -3,8 +3,22 @@ import {
 	type UserSignUpResponseDto,
 } from "~/modules/users/libs/types/types.js";
 import { type UserService } from "~/modules/users/user.service.js";
+import { type AuthService as AuthServiceT } from "./libs/types/types.js";
+import {
+	AuthError,
+	HTTPCode,
+	type UserWithPassword,
+	type UserSignInRequestDto,
+	type UserSignInResponseDto,
+} from "shared";
+import { cryptCompare } from "./helpers/crypt/crypt-compare.helper.js";
+import { ExceptionMessage } from "~/libs/enums/enums.js";
+
+function createTokenStub({ id }: { id: number }) {
+	return `token for user with id=${id}`;
+}
 
-class AuthService {
+class AuthService implements AuthServiceT {
```

Comment:
```suggestion
class AuthService {
```
No need to create AuthService type